app/javascript/css/styles.scss

```diff
@@ -43,6 +52,85 @@
     color: #363636;
 }
 
+
+.field-name .field-email .field-surname .field-password{
+    display: flex;
+    flex-direction: row;
+}
+.field_with_errors { 
+    display: inline;
+}
+ 
+.field_with_errors input {
+    padding: 3px 2px;
+    border: 1.5px solid #d93025;
+}
+
+select {
+    width: 274px;
+    height: 45px;
+    margin: 6px 0 0;
+    padding: 12px 10px 12px 13px;
+    border-radius: 5px;
+    border: solid 1px #e0e7ff;
+    background-color: rgba(224, 231, 255, 0.2);
+    font-size: 15px;
+    font-family: Lato;
+}
+.required-email {
+    color: #9e9e9e;
```

Comment:
instead of repeating the same styles of each class I suggest to use a common class for the different divs. I think 'label-email' is already set and could do the job